Doc/license.rst

```diff
@@ -22,7 +22,7 @@ Virginia where he released several versions of the software.
 In May 2000, Guido and the Python core development team moved to BeOpen.com to
 form the BeOpen PythonLabs team.  In October of the same year, the PythonLabs
 team moved to Digital Creations (now Zope Corporation; see
-https://www.zope.org/).  In 2001, the Python Software Foundation (PSF, see
+https://www.zope.dev/).  In 2001, the Python Software Foundation (PSF, see
```

Comment:
This one should be ignored rather than changed!

Reply:
Agreed, the reason is we're extra cautious about touching the licence file and prefer not to touch it at all if possible :)

Reply:
Thanks for the input. I wasn't aware of that. Will add to allowed redirect

Reply:
Done. I undid the change to Doc/license.rst, and Doc/conf.py now has an entry in allowed redirect.